lib/metro-transit.js

```diff
@@ -57,12 +57,14 @@ MetroTransit.prototype.intentHandlers = {
         }, {});
 
         console.info('Train Arrivals', trainArrivals);
-        if (_.size(trainArrivals) === 0) {
-          response.tell(props.stationNoArrivalsSpeechOutput);
+        var responseText = utils.makeGetStationResponseText(trainArrivals, stationName);
+        // if there's <= 1 train arriving we don't need any more info from the user; we can just tell them when that
+        // train is coming or that no trains are coming. Otherwise we're going to need to respond w/ a question.
+        if (_.size(trainArrivals) <= 1) {
```

Comment:
nit: to me, it makes more sense to reverse this, so the condition is if > 1 ask for more info, else return because you know the answer.